### PR TITLE
Cat Ears and Tail Support for Humans

### PR DESCRIPTION
**Pull request by DieselMohawk**
## About the PR
 Makes Cat markings selectable in the loadout menu

## Why / Balance
 We have cat fellas who would appreciate it.

## Technical details
 
``Resources/Prototypes/Species/human.yml`` Adds Tail Layer to character Human species and makes cat ears and tail have a marking token

## Media

Screenshot (4) 
Screenshot (5) 
Screenshot (6)

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Made Cat Markings Accessible

### CODE REVIEW

**Comment by FluffMe** [comment]
Tagging as needing admin discussion, as my budget felinids were closed once.

**Comment by RotEmpress** [comment]
From my understanding, I'm sure we can get all the admins to discuss this equally. I'm pretty sure the reasoning as to why we did not allow for Fluffme's PR was because we did not see Harmony as something that was gonna stretch far balance-wise but seeing as we're slowly pulling away from that idea, while i'm not opposed to this merge OR FluffMe's Old PR. I feel like these are short-term solutions for a long-term thing we wanna implement here because at this point it feels like we just wanna add Felinids without them actually being Felinids, which feels counteractive, if the people want felinids. Just give 'em felinids, IMO.

I'd rather just see Felinids be implemented at this point but have them on a more grounded take. (This is Jajsha's Major Concern as well.) Whether they're from a Chemical/Gentical Explosion (If we're basing it off TG) gone completely wrong. To just humans having the potential of cloning errors which causes them, doesn't really matter to me. I am entirely down to do the majority of their spriting and making them a genuine species or if we wanna copy-left Delta V's code for them and deviate from there, feel free to bug me and If we want them to be an actual species or hell- a subspecies of human.

The major concern for them atleast from what I know of is that we just don't want them to be simplified and perfect cat-girls. We want them to fit the setting and feel like their plausible.

**Comment by DieselMohawk** [comment]
> From my understanding, I'm sure we can get all the admins to discuss this equally. I'm pretty sure the reasoning as to why we did not allow for Fluffme's PR was because we did not see Harmony as something that was gonna stretch far balance-wise but seeing as we're slowly pulling away from that idea, while i'm not opposed to this merge OR FluffMe's Old PR. I feel like these are short-term solutions for a long-term thing we wanna implement here because at this point it feels like we just wanna add Felinids without them actually being Felinids, which feels counteractive, if the people want felinids. Just give 'em felinids, IMO.
> 
> I'd rather just see Felinids be implemented at this point but have them on a more grounded take. (This is Jajsha's Major Concern as well.) Whether they're from a Chemical/Gentical Explosion (If we're basing it off TG) gone completely wrong. To just humans having the potential of cloning errors which causes them, doesn't really matter to me. I am entirely down to do the majority of their spriting and making them a genuine species or if we wanna copy-left Delta V's code for them and deviate from there, feel free to bug me and If we want them to be an actual species or hell- a subspecies of human.
> 
> The major concern for them atleast from what I know of is that we just don't want them to be simplified and perfect cat-girls. We want them to fit the setting and feel like their plausible.

Currently I'm attempting this PR as it's a means of greater expression for a good amount of people who currently lack alternatives to felinids due to other rejected PRs. I want this PR for until other, more impactful, PRs are accepted to fill this slot, at which point this PR will be reverted and the other PR will take its place. Yes, it's a short-term solution, and that's fine. This PR will be the metaphorical bench warmer while a more long-term solution is agreed upon.

**Comment by RotEmpress** [comment]
Apologies Branches, I meant to respond to Fluffme's response, didn't mean for it to be aimed at you!

**Comment by DieselMohawk** [comment]
> Apologies Branches, I meant to respond to Fluffme's response, didn't mean for it to be aimed at you!

Oh no, I just wanted to state my intentions.

**Comment by RotEmpress** [comment]
> > Apologies Branches, I meant to respond to Fluffme's response, didn't mean for it to be aimed at you!
> 
> Oh no, I just wanted to state my intentions.

Yeah, I get your intentions, as said before I got no issues with this PR personally but eventually I wanna see some long-term solutions. The only reason why I haven't made felinids for harmony is literally because I do not know how to code at all _otherwise you'd be my ass I'd be working on felinids faster than any other content that i've wanted to do._

**Comment by Zap527** [comment]
I've talked about this with admins a little bit but I don't think humans should have cat tails.

**Comment by DieselMohawk** [comment]
> I've talked about this with admins a little bit but I don't think humans should have cat tails.

Can I get a concensus for this? I can remove it if need be, it sorta doesn't look that good anyways.

**Comment by TheCrimsonJupiter** [comment]
> > I've talked about this with admins a little bit but I don't think humans should have cat tails.
> 
> Can I get a concensus for this? I can remove it if need be, it sorta doesn't look that good anyways.

I'm fine with this being merged as is, but I agree with Rot_ that actually adding Felinids would make more sense now that licenses have changed. I think Felinids scaled to human size would be a pretty good solution for Harmony.

**Comment by RotEmpress** [comment]
> > > I've talked about this with admins a little bit but I don't think humans should have cat tails.
> > 
> > 
> > Can I get a concensus for this? I can remove it if need be, it sorta doesn't look that good anyways.
> 
> I'm fine with this being merged as is, but I agree with Rot_ that actually adding Felinids would make more sense now that licenses have changed. I think Felinids scaled to human size would be a pretty good solution for Harmony.

Size needs to be same size as humans, IMO. Otherwise, I'm not personally a fan of hairballs either but we can talk more in-depth of what we do or don't want later on when their actually being implemented.

**Review by spanky-spanky — APPROVED** [review]

File: Resources/Prototypes/Species/human.yml [hunk 1]
```diff
@@ -33,12 +33,13 @@
     RLeg: MobHumanRLeg
     LFoot: MobHumanLFoot
     RFoot: MobHumanRFoot
+    Tail: MobHumanoidAnyMarking
```
Comment (FluffMe) [inline Resources/Prototypes/Species/human.yml:36]: Missing harmoy comment

**Review by FluffMe — CHANGES_REQUESTED** [review]

**Review by KeldWolf — DISMISSED** [review]
*cue From Software intro about the age of the cat ears beginning* 

Sounds like we're green light for this so I approve.

**Comment by KeldWolf** [comment]
Obviously after FluffMe's requested changes.

File: Resources/Prototypes/Species/human.yml (hunk 1 above)
Comment (DieselMohawk) [reply Resources/Prototypes/Species/human.yml:36]: Whoops

**Review by FluffMe — APPROVED** [review]

**Pull request merged**